source/features/keyboard-shortcuts.js

```diff
@@ -0,0 +1,55 @@
+const toggleNightMode = () => {
+	if (document.querySelector('.nightmode-toggle')) {
+		document.querySelector('.nightmode-toggle').click();
+	}
+};
+
+export default async () => {
+	const customShortcuts = [
+		{
+			name: 'Actions',
+			description: 'Shortcuts for common actions.',
+			shortcuts: [
+				{
+					keys: [
+						'Ctrl',
+						'm'
+					],
+					description: 'Toggle Night Mode'
+				}
+			]
+		},
+		{
+			name: 'Navigation',
+			description: 'Shortcuts for navigating between items in timelines.',
+			shortcuts: []
+		},
+		{
+			name: 'Timelines',
+			description: 'Shortcuts for navigating to different timelines or pages.',
+			shortcuts: []
+		}
+	];
+
+	document.addEventListener('keydown', event => {
+		const keyName = event.key;
+		switch (keyName) {
+			case event.ctrlKey && 'm':
+				toggleNightMode();
+				break;
+			default:
+				break;
+		}
+	});
+
+	const initDataElement = document.querySelector('#init-data');
+	if (initDataElement) {
+		const initData = JSON.parse(initDataElement.value);
+		const updatedShortcuts = initData.keyboardShortcuts;
```

Comment:
Make sure you spread this, so you're working with a copy instead of a reference:

```javascript
const updatedShortcuts = [...initData.keyboardShortcuts];
```